### PR TITLE
Sl03mirebalais

### PR DESCRIPTION
**Pull request by lebbiesahr**
Sets up occupation lists in SectionsSierraLeone and prepares for later addition of Ebola screening questions

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsSierraLeone.java [hunk 1]
```diff
@@ -39,24 +58,26 @@ public Question getOccupationQuestion() {
...
+        w.getConfig().addOption("PIH:Cowherd", "zl.registration.patient.occupation.cowherd.label");
         w.getConfig().addOption("PIH:DRIVER", "zl.registration.patient.occupation.driver.label");
         w.getConfig().addOption("PIH:FACTORY WORKER", "zl.registration.patient.occupation.factoryWorker.label");
         w.getConfig().addOption("PIH:FARMER", "zl.registration.patient.occupation.farmer.label");
         w.getConfig().addOption("CIEL:159674", "zl.registration.patient.occupation.fisherman.label");
         w.getConfig().addOption("PIH:FRUIT OR VEGETABLE SELLER", "zl.registration.patient.occupation.fruitOrVegetableVendor.label");
         w.getConfig().addOption("PIH:HEALTH CARE WORKER", "zl.registration.patient.occupation.healthCareWorker.label");
         w.getConfig().addOption("PIH:1404", "zl.registration.patient.occupation.housework.label");
-        w.getConfig().addOption("PIH:HOUSEWORK/FIELDWORK", "zl.registration.patient.occupation.houseworkFieldwork.label");
+        w.getConfig().addOption("PIH:PIH:HOUSEWORK/FIELDWORK", "zl.registration.patient.occupation.houseworkFieldwork.label");
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsSierraLeone.java]: The double PIH looks like a typo?

Comment (lnball) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsSierraLeone.java]: * Fix "PIH:PIH:HOUSEWORK/FIELDWORK"

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsSierraLeone.java [hunk 2]
```diff
@@ -39,24 +58,26 @@ public Question getOccupationQuestion() {
 
         DropdownWidget w = new DropdownWidget();
 
-        // ordered alphabetically in English, with Unemployed and Other last
-        w.getConfig().addOption("PIH:BREWER", "zl.registration.patient.occupation.brewer.label");
+        // ordered alphabetically in Spanish, with Unemployed and Other last
```
Comment (lnball) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsSierraLeone.java]: You want this alphabetically in Spanish?

**Comment by lebbiesahr** [comment]
changes made and pushed

**Pull request merged**